android/app/src/main/java/com/votlin/android/ui/fragment/TalksListFragment.kt

```diff
@@ -41,56 +41,52 @@ class TalksListFragment : RootFragment<TalksListView>(), TalksListView {
     override val fragmentModule: Kodein.Module = Kodein.Module {
         bind<TalksListPresenter>() with provider {
             TalksListPresenter(
-                    view = this@TalksListFragment,
-                    errorHandler = AndroidErrorHandler(),
-                    executor = instance(),
-                    repository = instance())
+                view = this@TalksListFragment,
+                errorHandler = AndroidErrorHandler(),
+                executor = instance(),
+                repository = instance()
+            )
         }
     }
 
-    private val adapter = TalksAdapter { presenter.onTalkClicked(it) }
+    private lateinit var talksAdapter: TalksAdapter
 
     override fun initializeUI() {
-        talks.layoutManager = LinearLayoutManager(activity, LinearLayoutManager.VERTICAL, false)
-        talks.adapter = adapter
+        talksAdapter = TalksAdapter { presenter.onTalkClicked(it) }
+        talks.apply {
+            layoutManager = LinearLayoutManager(activity, LinearLayoutManager.VERTICAL, false)
+            adapter = talksAdapter
+        }
     }
 
     override fun registerListeners() {
         // Nothing to do here
     }
 
-    override fun showProgress() = progressView.showMe()
+    override fun showProgress() {
+        progressView.showMe()
+    }
 
-    override fun hideProgress() = progressView.hideMe()
+    override fun hideProgress() {
+        progressView.hideMe()
+    }
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
-        if (userVisibleHint) {
-            presenter.onTrackChanged(getTrack())
-        }
-    }
-
-    override fun setUserVisibleHint(isVisibleToUser: Boolean) {
-        super.setUserVisibleHint(isVisibleToUser)
-        if (isVisibleToUser && isResumed) {
-            presenter.onTrackChanged(getTrack())
-        }
+        presenter.onViewCreated(getTrack())
```

Comment:
In my opinion, it's a bad idea to put the presenter methods names like android lifecycle methods, think about that, in JS or iOS there is no method called "onViewCreated"

Reply:
sure, any suggestion for this name?